python-avd/pyavd/_eos_designs/structured_config/underlay/prefix_lists.py

```diff
@@ -93,34 +83,27 @@ def prefix_lists(self: AvdStructuredConfigUnderlayProtocol) -> list | None:
                 ):
                     sequence_number += 10
```

Comment:
use the enumerate and index like everywhere else to harmonize the code

Reply:
We have two for loops here, changing the equation is generating repeated sequence_number. 
If we use both loops indexes then we will get diff in cfg. (sequence_number = sequence_number + 10 * (index_peer + index_uplink +1))